src/tests/3-corax+.8o

```diff
@@ -32,13 +32,22 @@
 :macro drawop A B {
   i := A
   sprite x0 y 4
+
   i := B
   sprite x1 y 4
 }
 
 : test-2X-0E
   v0 := 1
   return
+
+  # These opcodes should only run if the interpreter doesn't have 00EE implemented.
+
+  # Note by Madster: I have seen one interpreter, of which I've forgotten the
+  # name of, that allows the execution of the opcode succeeding the return. There 
+  # is a chance that this would fail on that interpreter; however, I'll need to dig up that 
+  # interpreter to test that, though.
```

Comment:
Then that interpreter would be broken, and this test would successfully catch that issue ;)